apollo-portal/src/test/java/com/ctrip/framework/apollo/portal/component/UserIdentityContextHolderTest.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2025 Apollo Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.ctrip.framework.apollo.portal.component;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class UserIdentityContextHolderTest {
+
+  @BeforeEach
+  public void setUp() {
+    // Clear ThreadLocal before each test
+    UserIdentityContextHolder.setAuthType(null);
+    // Clear any existing values before each test
+    UserIdentityContextHolder.clear();
```

Comment:
Do we need to call both `UserIdentityContextHolder.setAuthType(null)` and `UserIdentityContextHolder.clear()`, or is `UserIdentityContextHolder.clear()` alone sufficient?